src/angular-app/languageforge/lexicon/editor/editor.component.html

```diff
@@ -41,14 +41,14 @@
                 </div>
             </div>
         </div>
-        <div class="row">
+        <div class="row" data-ng-show="$ctrl.isAtEditorEntry()">
```

Comment:
It is necessary to hide this div because otherwise it sits empty on the main list view taking up valuable height and also causing inconsistent behavior depending on if the editor has been opened previously or not.  This change provides consistent behavior.

Reply:
👍🏻